### ISSUE TITLE
Leftwm fucks up custom layout (Neo 2.0)

### ISSUE THREAD

**Issue by Septias** [leftwm/leftwm#1088]
### Describe the bug

When swiching to leftwm, my custom keyboard (Neo 2.0) got deactivated and now it is even broken on gnome. 
I tried leftwm about half a year ago, but already then there were some problems with my custom keyboard layout which is why I didn't switch back then. After some time I decided to try it again, but now it is even worse because it fucks up my gnome keyboartd setup. Even after removing/adding the layout again, after each restart I have to repeate that step to get my custom keyboard layout working in _windows_ again. On basic gonome it is activated, but as soon as I type in chrome or vs-code its still using normal QWERTZ.

Do you know a fix to that or can at least tell me which files are touched by the window manager that could fuck up the input device?

### OS / Distro

Linux <name> 5.19.17-2-MANJARO #1 SMP PREEMPT_DYNAMIC Sun Nov 6 00:08:27 UTC 2022 x86_64 GNU/Linux

### LeftWM Check

```text
will add later
```

**Comment by Eskaan** [comment]
You might want to look at #1063

**Comment by VuiMuich** [comment]
Could you please share ho you set your layout?
Have you modified your config to match with your layout?

Tbh. `leftwm` screwing up keylayouts for gnome sounds quite improbable.
Is it possible that this overlaps with some other system event like installing updates?

**Comment by VuiMuich** [comment]
Hey @Septias did you find a fix?
It would be nice if you could share a little info why you closed this now.

**Comment by Septias** [comment]
It is no longer relevant for me as I made a fresh OS-reinstall with Nix and now it works.

**Comment by VuiMuich** [comment]
I see and just saw your other issue about Alacritty now.

As I moved to NixOS recently and use a custom variant of the neo-family as well, I'd like to as you a thing not related to leftwm:
I had an issue that on layer 3 backspace was not working, only after adding `neo_qwertz` as secondary layout (will keep it probably around anyways in case someone else needs to type on that machine 😁) that worked.
Would you mind to share your experience and possibly your symbols file?
Also how are you adding your custom layout to your nixos config? I was thinking about actually patching it into the `de` symbols as in nixos it's not an issue of an update breaking this with an update of xkeyboard-config.
